tests/e2e/cluster_downgrade_test.go

```diff
@@ -76,6 +78,8 @@ func testDowngradeUpgrade(t *testing.T, clusterSize int) {
 		})
 	}
 	t.Logf("Cluster created")
+	generateSnapshot(t, 10, epc)
```

Comment:
Please use names const variables instead of magical numbers `10`

Reply:
updating.